Wrappers/Python/cil/framework/BlockDataContainer.py

```diff
@@ -139,76 +139,62 @@ def add(self, other, *args, **kwargs):
         :param: out (optional): provides a placehold for the resul.
         '''
         out = kwargs.get('out', None)
```

Comment:
none of these are used, right?

```suggestion
```

Reply:
Yep makes sense, looks like this is resolved now